metricbeat/module/rabbitmq/connection/data.go

```diff
@@ -30,14 +30,15 @@ import (
 
 var (
 	schema = s.Schema{
-		"name":        c.Str("name"),
-		"vhost":       c.Str("vhost", s.Required),
-		"user":        c.Str("user", s.Required),
-		"node":        c.Str("node", s.Required),
-		"channels":    c.Int("channels"),
-		"channel_max": c.Int("channel_max"),
-		"frame_max":   c.Int("frame_max"),
-		"type":        c.Str("type"),
+		"name":                 c.Str("name"),
+		"client_provided_name": c.Str("client_properties.connection_name"),
```

Comment:
I suggest to call this `client_provided.name` or event something like `client.name`. The reason is that probably we can enrich this with more fields in a near future (or maybe not) an, if so, we probably won't like to have
`client_provided_name`
`client_provided_tag`
`client_provided_label`

Reply:
Take into account that this is a name for the connection, so +1 to something like `client_provided.name`, but don't use `client.name`, it would look like the name of the client.
We may also keep the original `client_properties` naming, so if there are more properties in the future we store them as:
```
rabbitmq.connection.client_properties.name
rabbitmq.connection.client_properties.tag
...
```

Reply:
Yeah good point! Thanks @jsoriano and @sayden! I think I'm leaning towards using `rabbitmq.connection.client_provided.name` just to make it more consistent with the rabbitmq UI 😬